react/validateAddress.js

```diff
@@ -239,22 +325,33 @@ function validateCountry(value) {
   return validResult
 }
 
+/**
+ * @argument value {number[] | undefined}
+ *
+ * @returns {ValidationResult}
+ */
 function validateGeoCoordinates(value) {
```

Comment:
Why not renaming the file to `.ts` and using this:
```suggestion
function validateGeoCoordinates(value: number[] | undefined ): ValidationResult {
```

I think you explained to me before, but I don't remember very well :(

Reply:
because I would need to convert this whole file to typescript in one go :-/

it would probably mess up the git blame too (way more than this commit does already haha)

Reply:
> because I would need to convert this whole file to typescript in one go :-/

Doesn't typescript have a way to slowly convert JS to TS without the need to convert everything at once? Or that would require some extra effort as well?

Reply:
apparently it requires to do a bit more than what I did, because when converting the files directly to typescript builder-hub will start type checking them as well. So I had to add the `noImplicitAny: false` to the tsconfig (which isn't _that_ bad considering the whole repo is javascript for now) to get the build passing, and had to tweak some other functions I didn't before

Reply:
but other than that I got it working written in pure typescript, will push the commit shortly